package.json

```diff
@@ -22,6 +22,9 @@
     "newman": "./bin/newman.js"
   },
   "preferGlobal": true,
+  "files": [
+		"*",
```

Comment:
I think strict JSON parsing doesn't like this trailing comma and that's why the checks are failing.

Reply:
Thanks . Done .